gem/client/css/variables/_variables.scss

```diff
@@ -35,5 +35,14 @@ $colorWhiteSmoke: #ececec;
 $colorPurple: #7300FF;
 $colorYellow: #FFFC80;
 $colorTurq: #A4EED2;
-$colorOrange: #FCAB88; 
-$colorPink: #FF87D9; 
+$colorOrange: #FCAB88;
```

Comment:
not necessarily for this PR, but isn't it problematic to have these types of classes? eventually someone is going to do this: `$colorWhite: #000` 😬